cmd/neblictl/internal/controlplane/executors.go

```diff
@@ -270,7 +270,7 @@ func (e *Executors) StreamsCreate(ctx context.Context, parameters interpoler.Par
 		// Propagate new configuration
 		err := e.controlPlaneClient.setSamplerConfig(ctx, resourceAndSamplerEntry.sampler, resourceAndSamplerEntry.resource, update)
 		if err != nil {
-			writer.WriteStringf("%s.%s: Could not create the stream because %v\n", resourceAndSamplerEntry.resource, resourceAndSamplerEntry.sampler, err)
+			writer.WriteStringf("%s.%s: ERROR %v\n", resourceAndSamplerEntry.resource, resourceAndSamplerEntry.sampler, err)
```

Comment:
This significantly changes the error message format, we do not have a "standard" defined but putting the error level in upper case only in a few cases makes these seem like they are specially bad errors. All changes to the error messages should be consistent and I probably deserve their own PR

Reply:
In case of error, neblictl showed something like `kafka-sampler.order-confirmation: Could not create the stream because error configuring the rule rules: invalid %s uid %s, expected alphanumerical with spaces and ./() character`. Half of the error is boilerplate that does not provide any useful information. In fact, it's misleading, it talks about rules, when it does not have anything to do with it. As different kind of errors can happen, and in multiple contexts (digests, streams and events), I thought it would be simpler to show the inner error without adding boilerplate.